_includes/parse-server/adapters.md

```diff
@@ -0,0 +1,7 @@
+# Adapters
+
+All official adapters are distributed as scoped pacakges on [npm (@parse)](https://www.npmjs.com/search?q=scope%3Aparse).
+
+Some well maintained adapters are also available on the [Parse Server Modules](https://github.com/parse-server-modules) organization.
```

Comment:
We ought to get any of these which aren't in the main org transferred over and close this out. Arthur has access, maybe someone else does but is set to private cc @davimacedo @dplewis.

They do all appear to be unmaintained and on the surface not well used.